### ISSUE TITLE
Zoom 17 missing some building parts

### ISSUE THREAD

**Issue by nvkelso** [tangrams/eraser-map#187]
_Migrating this from https://github.com/tilezen/vector-datasource/issues/912..._
# tl;dr

From [this](https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L2492-L2500) chunk of code, sleuthed by @zerebubuth:

```
                    - { $zoom: [16], height: { min: 100 } }
                    - { $zoom: [16], area: { min: 3500 } }
                    - { $zoom: [16], volume: { min: 50000 } }
                    - { $zoom: [17], area: { min: 500 } }
                    - { $zoom: [17], volume: { min: 15000 } }
                    - { $zoom: { min: 18 } }
```

We think something like this needs to be added to zoom 17 filter set:

```
                    - { $zoom: [17], height: { min: 100 } }
```
# Mike's original report

Go to: http://tangrams.github.io/bubble-wrap/#18.87804/40.74546/-73.99049

Note bump on southern side of the building below the waldy's woodfire pizza label: 

![screen shot 2016-07-20 at 5 09 49 pm](https://cloud.githubusercontent.com/assets/307758/17003217/db76455e-4e9c-11e6-8fc8-69cb5c90609e.png)

Zoom out once: http://tangrams.github.io/bubble-wrap/#17.87804/40.74546/-73.99049

Note that the bump is gone:
![screen shot 2016-07-20 at 5 11 37 pm](https://cloud.githubusercontent.com/assets/307758/17003246/0af30ad8-4e9d-11e6-98b9-8c3e3c6a4927.png)

Zoom out again: http://tangrams.github.io/bubble-wrap/#16.87804/40.74546/-73.99049

The bump has returned. 
![screen shot 2016-07-20 at 5 12 15 pm](https://cloud.githubusercontent.com/assets/307758/17003268/20202a9e-4e9d-11e6-8261-0c8fc7bf6088.png)

This happens on web and on ES
